x-pack/platform/plugins/shared/streams/server/lib/streams/state_management/streams/classic_stream.ts

```diff
@@ -221,6 +223,21 @@ export class ClassicStream extends StreamActiveRecord<Streams.ClassicStream.Defi
       }
     }
 
+    if (this._changes.processing && this._definition.ingest.processing.steps.length > 0) {
+      try {
+        validateStreamlang(this._definition.ingest.processing, 'classic');
```

Comment:
Should we do this? On my validation PR I didn't validate this kind of stuff for classic streams because I didn't want to break existing usages and most of the stuff we validate is only relevant for wired streams anyway